scripts/Muon/GUI/MuonAnalysis/muon_analysis_2.py

```diff
@@ -299,8 +300,5 @@ def setup_fitting_notifier(self):
             self.dockable_plot_widget.presenter.plot_guess_observer)
 
     def closeEvent(self, event):
-        self.tabs.closeEvent(event)
```

Comment:
these lines are being deleted are what caused the closing problem @Mantid-Matthew saw. To ensure all of the ejected tabs and plot have been closed, we manually override the close signal for the main window. It then calls these methods which force close each tab (even if it has been removed). 

We normally don't have to do this because as a child qt will automatically close it for us. However, when we undock it the tab/plot lose the parent and can therefore hang around after the main GUI closes.

Reply:
@AnthonyLim23 I'm confused we're both pointing to the same lines being removed in the same file.

Reply:
We are, you put at the end of the block and I put it at the start. On the upside its obvious what to put back into the code.